app/client/lib/ethereum/observeCustomContracts.js

```diff
@@ -7,15 +7,15 @@ var addLogWatching = function(newDocument){
     if(blockToCheckBack < 0)
         blockToCheckBack = 0;
 
-    // console.log('EVENT LOG:  Checking Custom Contract Events for '+ newDocument.address +' (_id: '+ newDocument._id + ') from block # '+ blockToCheckBack);
+    console.log('EVENT LOG:  Checking Custom Contract Events for '+ newDocument.address +' (_id: '+ newDocument._id + ') from block # '+ blockToCheckBack);
```

Comment:
it seems you are uncommenting a lot of code that is not related to that specific fix, but it's about event logging